`test_import_sorting.py` failures when run from outside the source tree

**Describe the bug**
When `pytest` is run from outside the Darker source tree, some tests fail.

**To Reproduce**
Steps to reproduce the behavior:
1. Clone Darker: `git clone https://github.com/akaihola/darker`
2. In a fresh virtualenv, install the cloned Darker with test dependencies: `pip install './darker[test]'`
3. Run tests with `pytest ./darker/src/darker/tests/test_import_sorting.py`
4. Observe test failures

**Expected behavior**
No test failures

**Screenshots**
```python
pytest ./darker/src/darker/tests/test_import_sorting.py
================================================ test session starts ================================================
platform linux -- Python 3.11.3, pytest-7.3.1, pluggy-1.0.0
rootdir: /tmp/darker
configfile: pytest.ini
plugins: darker-0.1.2, kwparametrize-0.0.3
collected 64 items                                                                                                  

darker/src/darker/tests/test_import_sorting.py ..............................................F.F............. [ 96%]
..                                                                                                            [100%]

===================================================== FAILURES ======================================================
___________________________________ test_build_isort_args[src0-None-None-expect0] ___________________________________

src = PosixPath('file.py'), config = None, line_length = None, expect = {'settings_path': '/tmp'}

    @pytest.mark.kwparametrize(
        dict(src=Path("file.py"), expect={"settings_path": "{cwd}"}),
        dict(
            config="myconfig.toml",
            expect={"settings_file": "myconfig.toml"},
        ),
        dict(line_length=42, expect={"settings_path": "{cwd}", "line_length": 42}),
        src=Path("file.py"),
        config=None,
        line_length=None,
    )
    def test_build_isort_args(src, config, line_length, expect):
        """``_build_isort_args`` returns correct arguments for isort"""
        result = darker.import_sorting._build_isort_args(src, config, line_length)
    
        if "settings_path" in expect:
            expect["settings_path"] = str(expect["settings_path"].format(cwd=Path.cwd()))
>       assert result == expect
E       AssertionError: assert {'settings_path': '/'} == {'settings_path': '/tmp'}
E         Differing items:
E         {'settings_path': '/'} != {'settings_path': '/tmp'}
E         Use -v to get more diff

darker/src/darker/tests/test_import_sorting.py:184: AssertionError
____________________________________ test_build_isort_args[src2-None-42-expect2] ____________________________________

src = PosixPath('file.py'), config = None, line_length = 42, expect = {'line_length': 42, 'settings_path': '/tmp'}

    @pytest.mark.kwparametrize(
        dict(src=Path("file.py"), expect={"settings_path": "{cwd}"}),
        dict(
            config="myconfig.toml",
            expect={"settings_file": "myconfig.toml"},
        ),
        dict(line_length=42, expect={"settings_path": "{cwd}", "line_length": 42}),
        src=Path("file.py"),
        config=None,
        line_length=None,
    )
    def test_build_isort_args(src, config, line_length, expect):
        """``_build_isort_args`` returns correct arguments for isort"""
        result = darker.import_sorting._build_isort_args(src, config, line_length)
    
        if "settings_path" in expect:
            expect["settings_path"] = str(expect["settings_path"].format(cwd=Path.cwd()))
>       assert result == expect
E       AssertionError: assert {'line_length...gs_path': '/'} == {'line_length...path': '/tmp'}
E         Omitting 1 identical items, use -vv to show
E         Differing items:
E         {'settings_path': '/'} != {'settings_path': '/tmp'}
E         Use -v to get more diff

darker/src/darker/tests/test_import_sorting.py:184: AssertionError
================================================= warnings summary ==================================================
../home/kaiant/.virtualenvs/py311/lib/python3.11/site-packages/_pytest/config/__init__.py:1302
  /home/kaiant/.virtualenvs/py311/lib/python3.11/site-packages/_pytest/config/__init__.py:1302: PytestConfigWarning: Unknown config option: show_capture
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=============================================== slowest 10 durations ================================================
0.02s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content2-expect2-\n-utf-8]
0.02s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content2-expect2-\r\n-utf-8]
0.02s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content4-expect4-\n-utf-8]
0.01s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content2-expect2-\n-iso-8859-1]
0.01s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content4-expect4-\n-iso-8859-1]
0.01s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content1-expect1-\r\n-iso-8859-1]
0.01s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content3-expect3-\n-iso-8859-1]
0.01s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content3-expect3-\r\n-iso-8859-1]
0.01s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content3-expect3-\n-utf-8]
0.01s call     src/darker/tests/test_import_sorting.py::test_apply_isort[content2-expect2-\r\n-iso-8859-1]
============================================== short test summary info ==============================================
FAILED darker/src/darker/tests/test_import_sorting.py::test_build_isort_args[src0-None-None-expect0] - AssertionError: assert {'settings_path': '/'} == {'settings_path': '/tmp'}
FAILED darker/src/darker/tests/test_import_sorting.py::test_build_isort_args[src2-None-42-expect2] - AssertionError: assert {'line_length...gs_path': '/'} == {'line_length...path': '/tmp'}
====================================== 2 failed, 62 passed, 1 warning in 0.76s ======================================
```

**Environment (please complete the following information):**
 - OS: Linux / NixOS 21.11
 - Python version: 3.11.3
 - Darker version: 1.7.2

--- a/mypy.ini
+++ b/mypy.ini
@@ -60,11 +60,9 @@ disallow_subclassing_any = False
 [mypy-darker.tests.conftest]
 disallow_any_unimported = False
 
-[mypy-darker.tests.helpers]
-disallow_any_explicit = False
-
 [mypy-darker.tests.*]
 disallow_any_decorated = False
+disallow_any_explicit = False
 disallow_untyped_calls = False
 disallow_untyped_defs = False
 

--- a/src/darker/tests/test_import_sorting.py
+++ b/src/darker/tests/test_import_sorting.py
@@ -5,6 +5,7 @@
 from importlib import reload
 from pathlib import Path
 from textwrap import dedent
+from typing import Any, Dict, Optional
 
 import pytest
 
@@ -12,6 +13,7 @@ import darker.import_sorting
 from darker.git import EditedLinenumsDiffer
 from darker.tests.helpers import isort_present
 from darkgraylib.git import RevisionRange
+from darkgraylib.testtools.git_repo_plugin import GitRepoFixture
 from darkgraylib.utils import TextDocument, joinlines
 
 ORIGINAL_SOURCE = ("import sys", "import os", "", "print(42)")
@@ -171,7 +173,13 @@ def test_isort_config(monkeypatch, tmpdir, line_length, settings_file, expect):
     config=None,
     line_length=None,
 )
-def test_build_isort_args(src, config, line_length, expect):
+def test_build_isort_args(
+    git_repo: GitRepoFixture,
+    src: Path,
+    config: Optional[str],
+    line_length: int,
+    expect: Dict[str, Any],
+) -> None:
     """``_build_isort_args`` returns correct arguments for isort"""
     result = darker.import_sorting._build_isort_args(src, config, line_length)
 

